<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ajay-singh-lodhi" failures="3" tests="35" name="TestScript.CattlecallTest" time="888.973" errors="0" timestamp="2024-04-24T18:56:16 IST" skipped="0">
  <testcase classname="TestScript.CattlecallTest" name="Login" time="15.622"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Click_On_Cattle_Call" time="18.150"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Validate_Cattle_Call" time="21.312"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Click_On_Now_Button" time="23.497"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Validate_Now_Button" time="26.775"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Click_On_Cancel_Button_Without_Selecting_Users" time="26.612"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Click_On_Invite_Button_Without_Selecting_Users" time="27.295"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Search_User" time="28.015"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Enter_Invalid_Email_In_External_User" time="32.409"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Click_On_Invite_Button_With_Selecting_Users_External_User" time="34.433"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Edit_The_Cattle_Call_Meeting" time="25.675"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Edit_The_Cattle_Call_Meeting_With_Schedule_Button" time="27.404"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Edit_The_Cattle_Call_Meeting_With_Cancel_Button" time="25.946"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Edit_The_Cattle_Call_Meeting_With_Back_Button" time="25.845"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Delete_The_Cattle_Call_Meeting_With_No_Button" time="25.805"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Delete_The_Cattle_Call_Meeting_With_Close_Button" time="25.726"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Delete_The_Cattle_Call_Meeting_With_Yes_Button" time="26.800"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Click_On_Later_Button" time="23.181"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Schedule_The_Meeting_Without_Entering_Any_Data" time="25.628"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Schedule_The_Meeting_With_Entering_Only_Title_Name" time="26.958"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Schedule_The_Meeting_With_Title_Name_From_Time" time="28.024">
    <failure message="test case pass expected [12:45] but found [12:40]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: test case pass expected [12:45] but found [12:40]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at Pages.Cattlecallpage.Select_From_Time_In_Minute(Cattlecallpage.java:544)
at TestScript.CattlecallTest.Schedule_The_Meeting_With_Title_Name_From_Time(CattlecallTest.java:1107)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Schedule_The_Meeting_With_Title_Name_From_Time -->
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Schedule_The_Meeting_With_Title_Name_From_And_To_Time" time="28.490">
    <failure message="test case pass expected [12:45] but found [12:40]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: test case pass expected [12:45] but found [12:40]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at Pages.Cattlecallpage.Select_From_Time_In_Minute(Cattlecallpage.java:544)
at TestScript.CattlecallTest.Schedule_The_Meeting_With_Title_Name_From_And_To_Time(CattlecallTest.java:1174)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Schedule_The_Meeting_With_Title_Name_From_And_To_Time -->
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Enter_Details_And_Add_User" time="27.860">
    <failure message="test case pass expected [12:45] but found [12:40]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: test case pass expected [12:45] but found [12:40]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1577)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at Pages.Cattlecallpage.Select_From_Time_In_Minute(Cattlecallpage.java:544)
at TestScript.CattlecallTest.Enter_Details_And_Add_User(CattlecallTest.java:1248)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- Enter_Details_And_Add_User -->
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Validate_Today_Meeting" time="20.247"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Validate_Join_Meeting" time="20.824"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Click_On_Submit_Button_Without_Entering_Meeting_ID" time="24.763"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Click_On_Submit_Button_With_Entering_Invalid_Meeting_ID" time="24.627"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Click_On_Submit_Button_With_Entering_Valid_Meeting_ID" time="23.873"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Validate_Upcoming_Meeting" time="20.605"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Upcoming_Meeting" time="27.401"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Click_On_Schedule_Later" time="30.172"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Click_On_Meeting_Tab" time="22.736"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Click_On_Archives_In_Meeting_Tab" time="23.237"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Click_On_Cancel_Meeting" time="28.387"/>
  <system-out/>
  <testcase classname="TestScript.CattlecallTest" name="Click_On_Start_Meeting" time="24.639"/>
  <system-out/>
</testsuite> <!-- TestScript.CattlecallTest -->
